openshift-install-wrapper

```diff
@@ -80,9 +80,9 @@ verbose() { [[ ${VERBOSE} -eq 1 ]] && out "$@" || true; }
 # oc helper to run commands in a cluster
 oc() {
   if [[ ${VERBOSE} -eq 1 ]]; then
-    ${client} --kubeconfig ${clusterdir}/auth/kubeconfig $@
+    ${client} --kubeconfig ${clusterdir}/auth/kubeconfig "$@" -v=6
   else
-    ${client} --kubeconfig ${clusterdir}/auth/kubeconfig $@ &>/dev/null
+    ${client} --kubeconfig ${clusterdir}/auth/kubeconfig "$@"
```

Comment:
1. $@ should be quoted, without quotes lead to some issue in passing quoted strings.
2. Not having the output of the command is something not always wanted, have more sense to improve verbosity when required.

Reply:
I added in a later commit a nested if to check ${QUIET} to remove any kind of ouptut.